src/classes/UTIL_CascadeDeleteLookups_TDTM.cls

```diff
@@ -228,35 +230,64 @@ public abstract class UTIL_CascadeDeleteLookups_TDTM extends TDTM_Runnable {
                     }
                 }
 
-                String error = String.format(Label.CascadeDeletionError, new List<String>{parentName}) + '<br/><br/>';
-                for (String errType : cc.errors.get(parentId).keySet()) {
-                    error += ' ' + errType + ': ' + getErrorLinks(cc,cc.errors.get(parentId).get(errType));
+                List<String> errorTypeWithChildRecords = new List<String>();
+                for (String errorType : cc.errors.get(parentId).keySet()) {
```

Comment:
Could we make the variable name "cc" more descriptive? It works inside smaller scopes, e.g., loops but is not as clear as a parameter.